controllers/databasecluster_controller.go

```diff
@@ -1047,9 +1043,6 @@ func (r *DatabaseClusterReconciler) reconcilePXC(ctx context.Context, req ctrl.R
 	if database.Spec.Proxy.Type == "" {
 		database.Spec.Proxy.Type = everestv1alpha1.ProxyTypeHAProxy
 	}
-	if database.Spec.Proxy.Type == everestv1alpha1.ProxyTypeHAProxy && database.Spec.Proxy.Config == "" {
-		database.Spec.Proxy.Config = haProxyDefaultConfigurationTemplate
-	}
```

Comment:
Is the config required for the upstream operators to work correctly? Am I understand it correctly that If we don't set the Proxy config here then some default values from the upstream operators will be applied?

Reply:
ok, I see the description. resolving the question

Reply:
Yep. That's right the default configuration will be applied, however, we need to provide this option on the UI and in our documentation about changing defaults for proxies. The config is not required per se because all upstream operators have their own default configs.